src/Interfaces/Options/Interactivity/Modes/IOptionsInteractivityModePush.ts

```diff
@@ -0,0 +1,3 @@
+export interface IOptionsInteractivityModePush {
+    particles_nb: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'particles_nb' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)